docs/dev_guide.md

```diff
@@ -0,0 +1,60 @@
+# JupyterLab Developer Guide
+
+## Accessing JupyterLab Development Environment
+
+### 1. Create SSH Tunnel:
+
+Execute the following command to create an SSH tunnel to the remote server (`login1.berkeley.kbase.us`):
+
+```bash
+ssh -f -N -L localhost:44041:10.58.2.201:4041 <ac.anl_username>@login1.berkeley.kbase.us
+```
```

Comment:
I thought we already had docs like this somewhere?

Reply:
yea. Similar instructions in  the user's guide. https://github.com/kbase/cdm-jupyterhub/blob/main/docs/user_guide.md
But dev has different port number. That's also the reason I didn't make it as detailed as the user's guide.

Reply:
Maybe link to the user guide for more info?

Reply:
👍